core/src/main/java/org/lflang/generator/docker/DockerComposeGenerator.java

```diff
@@ -116,7 +100,53 @@ public void writeDockerComposeFile(List<DockerData> services, String networkName
     var contents =
         String.join(
             "\n", this.generateDockerServices(services), this.generateDockerNetwork(networkName));
-    FileUtil.writeToFile(contents, path);
-    context.getErrorReporter().nowhere().info(getUsageInstructions());
+    FileUtil.writeToFile(
+        contents, context.getFileConfig().getSrcGenPath().resolve("docker-compose.yml"));
+  }
+
+  /**
+   * Build using docker compose.
+   *
+   * @return {@code true} if successful,{@code false} otherwise.
+   */
+  public boolean build() {
+    return Objects.requireNonNull(
+                LFCommand.get(
+                    "docker",
+                    List.of("compose", "build"),
+                    false,
+                    context.getFileConfig().getSrcGenPath()))
+            .run()
+        == 0;
+  }
+
+  /** Create a launcher script that invokes Docker. */
+  public void createLauncher() {
+    var fileConfig = context.getFileConfig();
+    var packageRoot = fileConfig.srcPkgPath;
+    var srcGenPath = fileConfig.getSrcGenPath();
+    var file = fileConfig.binPath.resolve(fileConfig.name).toFile();
+    var script =
+        """
+        #!/bin/sh
+        set -euo pipefail
+        cd $(dirname "$0")
+        cd ..
```

Comment:
```suggestion
```
Do we need these two lines? `packageRoot` is an absolute path, right?

Reply:
We do need these because the used path `packageRoot.relativize(srcGenPath)`. Using a relative path is better because things won't break when you move the stuff to another location (e.g., within the filesystem or between machines).

Reply:
You're right about this and I'm OK with this as it is currently. One thing to be aware of however is that although the relative path has the advantage of allowing the directory to be moved, it has the disadvantage of not working when someone creates a symlink to the bin directory.

Reply:
I hadn't thought about that, but that seems like it might be an odd corner case? Let's not worry about it for now...